Documents

hossamBenhoud opened the pull request.

szalarhe committed:
compte rendu avancement groupes 05/10/16

szalarhe committed:
Update compteRendu.md

flodavid committed:
vidage, création de la branche
+ création des dossiers pour chaque groupe
+ ajout des documents de #architecture

flodavid committed:
Update README.md

alefol committed:
ajout options du compilateur

adrien3 committed:
Présentation et utilisation de Trello

flodavid committed:
Delete README.md

flodavid committed:
merging de organisation

flodavid committed:
Merge pull request #20 from ezlanguage/organisation

Organisation

Commit:
Add files via upload

Opérateur

NajiIbtissam committed:
Add files via upload

hossamBenhoud committed:
Version corrigé par M. Richer 

les fonctions ,les procedures, les conditions

NajiIbtissam committed:
Partie validée par M.Richer

alefol committed:
Create options_compilateur

Ascris committed:
Update Objectifs.md

flodavid committed:
Merge pull request #30 from ezlanguage/comp_doc_compilation

Update Objectifs.md

flodavid committed:
Update Objectifs.md

Commit:
Fonctions sur les chaînes de caractères

flodavid committed:
fichiers options compilation déplacés et markdown

hossamBenhoud committed:
Les classes EZL

Commit:
lang_fct_string : Fonctions sur les string validées

famzil committed:
Add files via upload

adrien3 committed:
Creation fichier vide pour l'installe de jenkins

flodavid committed:
Tutoriel git de Adrien au format pdf

flodavid commented:
Serait-il possible d'avoir un nom expliquant le but de la pull request et une description ? @hossamBenhoud

jdefaye committed:
Ajout fichier d'explication de la table des symboles

Ascris committed:
Présentation globale fonctionnement compilateur

Ascris committed:
Présentation globale fonctionnement compilateur

adrien3 committed:
Création de la documentation sur Jenkins, autour de l'installation d'un site PHP

adrien3 committed:
Merge pull request #80 from ezlanguage/orga_doc_jenkins

Document sur l'installation de Jenkins

Drissbenm committed:
Spécifications d'un module et programme

Spécifications d'un module, programme, inclusion des bibliothèques et inclusion du code cpp dans EZ
